src/Symfony/Bundle/FrameworkBundle/Command/AssetsPipelinePublishCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Asset\Pipeline\AssetPipeline;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Publishes the assets in the pipeline to the final output directory.
+ *
+ * This command is intended to be used during deployment.
+ *
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ *
+ * @final
+ */
+#[AsCommand(name: 'assets:pipeline:publish', description: 'Publishes all pipeline assets to the final public output directory.')]
+class AssetsPipelinePublishCommand extends Command
+{
+    public function __construct(
+        private readonly AssetPipeline $pipeline,
+        private readonly Filesystem $filesystem,
+        private readonly string $projectDir,
+        private readonly string $publicDirName,
+        private readonly bool $isDebug,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setHelp(<<<'EOT'
+The <info>%command.name%</info> command compiles and moves all the assets in
```

Comment:
I would say `compiles and dumps` rather than `compile and moves`. Moving would imply removing the source